src/sentry/api/endpoints/event_ai_suggested_fix.py

```diff
@@ -275,10 +275,7 @@ def suggest_fix(event_data, model=settings.SENTRY_AI_SUGGESTED_FIX_MODEL, stream
         temperature=0.7,
         messages=[
             {"role": "system", "content": prompt},
-            {
-                "role": "user",
-                "content": json.dumps(event_info),
-            },
+            dict(role="user", content=orjson.dumps(event_info).decode()),
```

Comment:
technically the dict literal is faster and should be preferred

Reply:
Yes, you're right. Jetbrains got me...